frontend/src/stories/PermissionsTable.stories.js

```diff
@@ -0,0 +1,247 @@
+        123: {'active': false,
+            'id': 123,
+            'name': 'Boot data sci 1 March 2021',
+            'permissions': ['TRUSTED_REVIEWER']},
+        124: {'active': false,
+            'id': 124,
+            'name': 'Boot web dev 1 March 2021',
+            'permissions': ['TRUSTED_REVIEWER']},
+        126: {'active': true,
+            'id': 126,
+            'name': 'Cohort 25 data eng',
+            'permissions': ['REVIEW_CARDS']},
+        127: {'active': true,
+            'id': 127,
+            'name': 'Cohort 25 data sci',
+            'permissions': ['REVIEW_CARDS']},
+        128: {'active': true,
+            'id': 128,
+            'name': 'Cohort 25 java',
+            'permissions': ['REVIEW_CARDS']},
+        129: {'active': true,
+            'id': 129,
+            'name': 'Cohort 25 web dev',
+            'permissions': ['REVIEW_CARDS']},
+        131: {'active': false,
+            'id': 131,
+            'name': 'Boot data eng 6 April 2021',
+            'permissions': ['TRUSTED_REVIEWER']},
+        132: {'active': false,
+            'id': 132,
+            'name': 'Boot web dev 6 April 2021',
+            'permissions': ['TRUSTED_REVIEWER']},
+        133: {'active': true,
+            'id': 133,
+            'name': 'Cohort 27 web dev',
+            'permissions': ['REVIEW_CARDS']},
+        134: {'active': true,
+            'id': 134,
+            'name': 'Provisionally accepted data sci',
+            'permissions': ['REVIEW_CARDS']},
+        135: {'active': true,
+            'id': 135,
+            'name': 'Provisionally accepted web dev',
+            'permissions': ['REVIEW_CARDS']},
+        136: {'active': true,
+            'id': 136,
+            'name': 'Provisionally accepted data eng',
+            'permissions': ['REVIEW_CARDS']},
+        137: {'active': true,
+            'id': 137,
+            'name': 'Provisionally accepted java',
+            'permissions': ['REVIEW_CARDS']},
+        139: {'active': true,
+            'id': 139,
+            'name': 'Cohort 25 web dev alumni',
+            'permissions': ['REVIEW_CARDS']},
+        140: {'active': true,
+            'id': 140,
+            'name': 'Cohort 25 java alumni',
+            'permissions': ['REVIEW_CARDS']},
+        141: {'active': true,
+            'id': 141,
+            'name': 'Cohort 25 data eng alumni',
+            'permissions': ['REVIEW_CARDS']},
+        142: {'active': true,
+            'id': 142,
+            'name': 'Cohort 25 web dev 1',
+            'permissions': ['REVIEW_CARDS']},
+        143: {'active': true,
+            'id': 143,
+            'name': 'Cohort 25 web dev 2',
+            'permissions': ['REVIEW_CARDS']},
+        144: {'active': true,
+            'id': 144,
+            'name': 'Boot web dev 3 May 2021',
+            'permissions': ['TRUSTED_REVIEWER']}}}
+    
+export const Primary = () => <PermissionsTable permissionInstances={permissionInstances}/>
```

Comment:
It would probably be better to jsust pass in the `permissionInstances.teams` because then the Presentation companent doesn't need to know as much about how to access the data.